presto-product-tests/README.md

```diff
@@ -0,0 +1,37 @@
+# Presto product tests
+
+## Test configuration
+
+Test assume that you have hadoop and presto cluster running. For setting up development environment we
```

Comment:
We introduced tests runner which drives tests execution. It is called outside the maven.